tests/test_pipelines.py

```diff
@@ -341,10 +343,74 @@ def test_tf_text_generation(self):
             nlp = pipeline(task="text-generation", model=model_name, tokenizer=model_name, framework="tf")
             self._test_mono_column_pipeline(nlp, VALID_INPUTS, {})
 
+    @slow
+    @require_torch
+    def test_integration_torch_dialogue(self):
+        # When
+        nlp = pipeline(task="dialogue", device=DEFAULT_DEVICE_NUM)
```

Comment:
Maybe pass `min_response_allowed_length` or (IMO clearer name `min_length_for_response`) here to test it.